test/integration/targets/inventory/runme.sh

```diff
@@ -88,3 +88,9 @@ fi
 
 # ensure we don't traceback on inventory due to variables with int as key
 ansible-inventory  -i inv_with_int.yml --list "$@"
+
+# test in subshell relative paths work mid play for extra vars in inventory refresh
+{
+	cd 1/2
+	ansible-playbook -e @../vars.yml -i inventory.yml 3/extra_vars_relative.yml "$@"
```

Comment:
If I understand things correctly, this command will not fail even if the fix is not present because failing to parse an inventory file is not fatal. Edits to the playbook itself I suggested above should rectify this, but we will need a small adjustment here:

```suggestion
	ansible-playbook -e @../vars.yml -i base_inventory.ini -i inventory.yml 3/extra_vars_relative.yml "$@"
```